recipes-debian/sudo/sudo_debian.bb

```diff
@@ -0,0 +1,44 @@
+#
+# base recipe: meta/recipes-extended/sudo/sudo_1.8.27.bb
+# base branch: warrior
+# base commit: fd73e062ea314deff3e247acaa5492b060c5155b
+#
+
+inherit debian-package
+require recipes-debian/sources/sudo.inc
+
+LICENSE = "GPL-2.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=eb723b61539feef013de476e68b5c50a"
```

Comment:
LICENSE and LIC_FILES_CHKSUM are  overwritten in recipes-extended/sudo/sudo.inc, do we need them here?

Reply:
No. I'm sorry. I removed it.